lib/pengine/utils.c

```diff
@@ -881,8 +881,10 @@ pe__failed_probe_for_rsc(const pcmk_resource_t *rsc, const char *name)
         rsc_id = pe__clone_child_id(parent);
     }
 
-    for (xmlNode *xml_op = pcmk__xe_first_child_any(rsc->cluster->failed);
+    for (xmlNode *xml_op = pcmk__xe_first_child(rsc->cluster->failed, NULL,
```

Comment:
One more chance to go back to the beautiful way that things once were lol... (with `pcmk__xe_first_child_any(xml)` and adding a new `pcmk__xe_first_child_named(xml, name)`)

Reply:
nope, too much to remember